packages/docusaurus-init/src/index.ts

```diff
@@ -182,23 +182,26 @@ export default async function init(
       : path.relative(process.cwd(), name);
 
   console.log();
-  console.log(`Success! Created ${chalk.cyan(cdpath)}`);
+  console.log(`Successfully created "${chalk.cyan(cdpath)}".`);
   console.log('Inside that directory, you can run several commands:');
   console.log();
   console.log(chalk.cyan(`  ${pkgManager} start`));
   console.log('    Starts the development server.');
   console.log();
   console.log(chalk.cyan(`  ${pkgManager} ${useYarn ? '' : 'run '}build`));
-  console.log('    Bundles the app into static files for production.');
+  console.log('    Bundles your website into static files for production.');
+  console.log();
+  console.log(chalk.cyan(`  ${pkgManager} ${useYarn ? '' : 'run '}serve`));
+  console.log('    Serve the built website locally.');
   console.log();
   console.log(chalk.cyan(`  ${pkgManager} deploy`));
-  console.log('    Publish website to GitHub pages.');
+  console.log('    Publish the website to GitHub pages.');
   console.log();
-  console.log('We suggest that you begin by typing:');
+  console.log('We recommend that you begin by typing:');
   console.log();
   console.log(chalk.cyan('  cd'), cdpath);
   console.log(`  ${chalk.cyan(`${pkgManager} start`)}`);
 
   console.log();
-  console.log('Happy hacking!');
+  console.log('Happy building awesome websites!');
```

Comment:
It's an attempt to distinguish from the [CRA](https://github.com/facebook/create-react-app/blob/fddce8a9e21bf68f37054586deb0c8636a45f50b/packages/react-scripts/scripts/init.js#L408), I think it's fine to have an original slogan.